### ISSUE TITLE
Question about z-axis value

### ISSUE THREAD

**Issue by eehoeskrap** [IDEA-Research/OSX#76]
I would like to know the range of the z-axis after calculating the smpl_x layer in the ubody dataset.
The smpl_x layey is as follows.

```
output = smpl_x.layer[gender](betas=shape, body_pose=body_pose.view(1, -1), global_orient=root_pose,
                                transl=trans, left_hand_pose=lhand_pose.view(1, -1),
                                right_hand_pose=rhand_pose.view(1, -1), jaw_pose=jaw_pose.view(1, -1),
                                leye_pose=zero_pose, reye_pose=zero_pose, expression=expr)


vertices = output.vertices.detach().numpy().squeeze() # 10475
joints = output.joints.detach().numpy().squeeze() # 144

```
For example, in smplx_annotation.json, the body_pose value of smplx_param is as follows.

[-0.1480163037776947, 0.011770674027502537, **0.08640401810407639**, 
-0.16378618776798248, -0.015610341913998127, **-0.1083071306347847**, 
0.2861928939819336, 0.00039016272057779133, **-0.006934669334441423**, 
...] 

However, after passing this value through the smpl_x layer, 144 joints are created, and the z-axis value becomes much larger than before.

[[0.0015399606199935079, 0.42420920729637146, **16.199426651000977**], 
[0.0608895868062973, 0.5198387503623962, **16.200016021728516**], 
[-0.06102701276540756, 0.5306458473205566, **16.196369171142578**], 
[-0.0005468808813020587, 0.32456669211387634, **16.253442764282227**], 
...]

**Comment by linjing7** [comment]
Hi, the z-axis value in `body_pose` represents the axis angle, while the z-axis value of the `joints` is the position. Thus, they are totally different concepts.

**Comment by mayank64ce** [comment]
What does 'global_orient' actually mean, I change the 3 values one-by-one and the output rendering changes but i dont exactly understand how ? I want to rotate the SMPL output but don't understand how.
